Sources/Todo.WebApi/Authorization/Policies.cs

```diff
@@ -0,0 +1,13 @@
+namespace Todo.WebApi.Authorization
+{
+    public static class Policies
+    {
+        public static class TodoItems
+        {
+            public const string CreateTodoItem = "create:todo";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/satrapu/aspnet-core-logging/pullRequest?prid=5148460)